clu/src/lib.rs

```diff
@@ -57,6 +61,26 @@ pub fn validate_mesh_ip(ip: &IpAddr) -> bool {
     ip.is_ipv6() && !ip.is_unspecified()
 }
 
+/// Performs some quick validation of the Address, mostly to make sure it's not junk
+pub fn validate_eth_address(address: &Address) -> bool {
+    // Special list of obviously invalid addresses that might be floating around
+    let list_of_junk_addresses = [
+        "0x0000000000000000000000000000000000000000"
+            .parse()
+            .unwrap(),
+        "0x0000000000000000000000000000000000000001"
+            .parse()
+            .unwrap(),
+        "0x0101010101010101010101010101010101010101"
+            .parse()
+            .unwrap(),
+    ];
+    if list_of_junk_addresses.contains(address) {
+        return false;
+    }
+    true
+}
```

Comment:
I don't think there's a way to really validate anything in this respect, all of those are valid Ethereum addresses that have a private key somewhere.

Reply:
this is mostly to remove the junk addresses we commonly use, there's no real validation possible otherwise, you're right about that.